railties/lib/rails/commands/rake_proxy.rb

```diff
@@ -0,0 +1,42 @@
+require 'rake'
+
+module Rails
+  module RakeProxy #:nodoc:
+
+    RAKE_TASKS_HELP_MESSAGE = <<-EOT
+In addition to those, you can run the rake tasks as rails commands:
+
+    EOT
+
+    private
+
+      def write_rake_tasks_help_message
+        puts RAKE_TASKS_HELP_MESSAGE
+      end
+
+      def write_rake_tasks
+        width = rake_tasks.map { |t| t.name_with_args.length }.max || 10
+        rake_tasks.each do |t|
+          printf("#{Rake.application.name} %-#{width}s  # %s\n", t.name_with_args, t.comment)
```

Comment:
Since I don't think we should print `Rake.application.name`, can we avoid setting it?